README.md

```diff
@@ -0,0 +1,27 @@
+# CytoCherryPick
+
+## Welcome to CytoCherryPick
+CytoCherryPick is an opensource sowftware that provides a set of tools for the selection of representative images from a set of images.
+
+## Functions of CytoCherryPick
+* **Image selection**: CytoCherryPick will select a representative image from a set of images.
+* **Image Distribution Analysis**: Choose your own representative image and CytoCherryPick will graph where in the image set distribution your image is located.
+
+# Installing and Using CytoCherryPick
+
+### Copy the repository
+``` bash
+$ git clone
+$ cd CytoCherryPick
+```
+
+### Install the CytoCherryPick env
```

Comment:
Consider completing the abbreviated `env` here as `environment` or defining it as an abbreviation somewhere before this line.